frontend/src/app/modules/boards/board/board-filter/board-filter.component.ts

```diff
@@ -8,19 +8,20 @@ import {IsolatedQuerySpace} from "core-app/modules/work_packages/query-space/iso
 import {QueryResource} from "core-app/modules/hal/resources/query-resource";
 import {HalResourceService} from "core-app/modules/hal/services/hal-resource.service";
 import {WorkPackageViewFiltersService} from "core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-filters.service";
-import {componentDestroyed, untilComponentDestroyed} from "ng2-rx-componentdestroyed";
 import {QueryFilterInstanceResource} from "core-app/modules/hal/resources/query-filter-instance-resource";
 import {UrlParamsHelperService} from "core-components/wp-query/url-params-helper";
 import {StateService} from "@uirouter/core";
 import {DebouncedEventEmitter} from "core-components/angular/debounced-event-emitter";
 import {skip} from "rxjs/internal/operators";
 import {ApiV3Filter} from "core-components/api/api-v3/api-v3-filter-builder";
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import {componentDestroyed} from "@w11k/ngx-componentdestroyed";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/modules/boards/board/board-filter/board-filter.component.ts#L19"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>